src/scss/3_components/form/form.scss

```diff
@@ -3,106 +3,275 @@
+    background-repeat: no-repeat, repeat;
+    background-position: right .7em top 50%, 0 0;
+    background-size: .65em auto, 100%;
+
+    @include breakpoint($phab, $no-query: true) {
+      font-size: 18px;
+      line-height: 1.5;
+      height: 2.5em;
+      padding: .5em 1.3334em .5em .6667em;
+    }
+  }
+
+  .form-select::-ms-expand {
+    display: none;
   }
-}
 
-// Based on https://css-tricks.com/styling-a-select-like-its-2019/
-.form-select {
-  display: block;
-  font-size: 16px;
-  font-family: sans-serif;
-  font-weight: normal;
-  color: $charcoal;
-  line-height: 1.3;
-  padding: .6em 1.4em .5em .8em;
-  box-sizing: border-box;
-  margin: rhythm(1) 0 0;
-  border: 1px solid $grey-dark;
-  border-radius: 0;
-  appearance: none;
-  background-color: #fff;
-  background-image: url('data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%22292.4%22%20height%3D%22292.4%22%3E%3Cpath%20fill%3D%22%23666666%22%20d%3D%22M287%2069.4a17.6%2017.6%200%200%200-13-5.4H18.4c-5%200-9.3%201.8-12.9%205.4A17.6%2017.6%200%200%200%200%2082.2c0%205%201.8%209.3%205.4%2012.9l128%20127.9c3.6%203.6%207.8%205.4%2012.8%205.4s9.2-1.8%2012.8-5.4L287%2095c3.5-3.5%205.4-7.8%205.4-12.8%200-5-1.9-9.2-5.5-12.8z%22%2F%3E%3C%2Fsvg%3E') linear-gradient(to bottom, #fff 0%, #e5e5e5 100%);
-  background-repeat: no-repeat, repeat;
-  background-position: right .7em top 50%, 0 0;
-  background-size: .65em auto, 100%;
-
-  @include breakpoint($phab, $no-query: true) {
-    font-size: 18px;
+  .form-select option {
+    font-weight: normal;
+  }
+
+  textarea,
+  .form-textarea {
+    width: 100%;
     line-height: 1.5;
-    height: 2.5em;
-    padding: .5em 1.3334em .5em .6667em;
   }
-}
 
-.form-select::-ms-expand {
-  display: none;
-}
+  .form-type-radio {
+    position: relative;
+  }
 
-.form-select:hover {
-  border-color: $brand-primary;
-}
+  input[type="checkbox"] {
+    border-radius: 0;
+  }
 
-.form-select:focus {
-  border-color: $brand-primary;
-  color: #222;
-  outline: none;
-}
+  input[type="radio"], .form-radio, input[type="checkbox"], .form-checkbox {
 
-.form-select option {
-  font-weight: normal;
-}
+    @include breakpoint($phab, $no-query: true) {
+      @include adjust-font-size-to(18px, 4);
+    }
+
+    @include adjust-font-size-to(16px, 4);
 
-textarea,
-.form-textarea {
-  width: 100%;
-  max-width: 620px;
-  line-height: 1.5;
+    // css trick to vertically center in parent div
+    position: absolute;
+    top: 50%;
+    transform: translate(0,-50%);
+    z-index: 1;
+    left: 1rem;
+    clear: both;
+    margin: 0;
+    padding: 0;
+    max-width: 2rem;
+    max-height: 2rem;
+    line-height: 2.4rem;
+    vertical-align: baseline;
+    box-shadow: none;
+    border: none;
+    color: inherit;
+
+    // sibling labels
+    & ~ label {
+      position: relative;
+      left: 0;
+      margin: 0.5em 0;
+      padding: 0.9rem 1.5rem 0.9rem 3rem;
```

Comment:
watch for the leading zero's on the padding values - these will flag on linting

Reply:
Bad habits die hard ...